src/nidigital/templates/session.py/fancy_get_pin_results_pin_information.py.mako

```diff
@@ -18,7 +18,7 @@
         pin_infos = []
         for i in range(len(pin_indexes)):
             pin_name = "" if pin_indexes[i] == -1 else self._get_pin_name(pin_indexes[i])
-            channel_name = self.get_channel_name(channel_indexes[i])
+            channel_name = self._get_channel_name(channel_indexes[i])
```

Comment:
Consider removing `_get_channel_name` and implementing this in term of `get_channel_names`. I think that's less code overall.